include/host-build.mk

```diff
@@ -44,6 +44,7 @@ ifneq ($(strip $(HOST_UNPACK)),)
   define Host/Prepare/Default
 	$(HOST_UNPACK)
 	$(Host/Patch)
+	[ ! -d ./src/ ] || $(CP) ./src/* $(HOST_BUILD_DIR)
```

Comment:
I haven't thought this through, but at the first glance I wonder why you evaluate that via negation?
Why not ``` [ -d ./src/ ] && $(CP)... ``` ?